plugins/modules/nmcli.py

```diff
@@ -1584,10 +1627,11 @@ def connection_options(self, detect_change=False):
         if self.mtu_conn_type:
             options.update({self.mtu_setting: self.mtu})
 
-        # Connections that can have a master.
-        if self.slave_conn_type:
+        # NetworkManager allows any interface type to be slave
```

Comment:
Have you tested using ```master``` with other interface types? It seem like we can stumble upon some bugs in future.

Reply:
I have tested it, but not all types. My bad.
Master is not acceptable to `ovs-bridge`. All another may be slave of bridge.
Future NM releases may have new connection types that not accept to be slave. I guess we come back to use `slave_conn_type` property that contains only types supported by module.
Thx for noticing.
I'll make changes soon.

Reply:
I returned back to using strict comparing with list of `master`-allowed types and add to it some widely used connection types.
More deeply testing shows that tunnel connections cannot be slaved to although NM allows to set `slave-type` and `master` to it. Definitely right way - is not accept thtis in module. I was too hasty doing it.
Fixed.